controllers/iochaos/podiochaosmanager/podiochaosmanager.go

```diff
@@ -60,8 +70,16 @@ func (m *PodIoManager) WithInit(key types.NamespacedName) *PodIoTransaction {
 	return t
 }
 
+// KeyErrorTuple is a tuple (Key, Err)
+type KeyErrorTuple struct {
```

Comment:
Good idea.

Actually, it's not a tuple. It's a tagged union, which could be an error with a key or a successful response with a key. The `tuple` is to simulate `tagged union` and the `struct` is used to simulate `tuple`... 

I agree with you that I should name it `CommitResponse` because `tuple` is also hard to understand here.